lib/supply/options.rb

```diff
@@ -33,19 +33,18 @@ def self.available_options
                                      optional: true,
                                      description: "Path to the directory containing the metadata files",
                                      default_value: (Dir["./fastlane/metadata/android"] + Dir["./metadata"]).first),
-        FastlaneCore::ConfigItem.new(key: :key,
-                                     env_name: "SUPPLY_KEY",
+        # REVIEW: better name than :json ? :jsonsecret ?
+        # REVIEW: what about migration from a previous config ? is there a way to auto-deprecate options ?
```

Comment:
As discussed, the plan is to keep the other options for now, but if they are detected we'll need to `UI.warn` that users should be using `json_key` instead.